openupgrade_scripts/scripts/account/17.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,497 @@
+---Models in module 'account'---
+obsolete model account.account.template
+obsolete model account.fiscal.position.account.template
+obsolete model account.fiscal.position.tax.template
+obsolete model account.fiscal.position.template
+obsolete model account.group.template
+obsolete model account.reconcile.model.line.template
+obsolete model account.reconcile.model.template
+obsolete model account.tax.repartition.line.template
+obsolete model account.tax.template
+# NOTHING TO DO: these models has been converted to CSV definition and existing company records XML-ID follow the same logic
+
+obsolete model account.invoice.send [transient]
+new model account.move.send [transient]
+# NOTHING TO DO: Replaced feature, but nothing to transfer, being transient models
+
+---Fields in module 'account'---
+account      / account.account          / _order                        : _order is now 'code, company_id' ('is_off_balance, code, company_id')
+account      / account.account          / is_off_balance (boolean)      : DEL
+# NOTHING TO DO: is_off_balance dissapears, but nobody will miss it, as it was underused and even incorrect (due to blocking certain operations)
+
+account      / account.account          / message_main_attachment_id (many2one): DEL relation: ir.attachment
```

Comment:
isn't it used to determine wich is the main attachement when they are multiple ? (such as the printed invoice on client invoice)